[WIP] 🌐 [i18n-KO] Translated `tasks/translation.mdx` to Korean

wonhyeongseo opened the pull request:
<!-- PR의 제목은 "🌐 [i18n-KO] Translated `<your_file>.mdx` to Korean" 으로 부탁드립니당 -->
# What does this PR do?

Translated the `tasks/translation.mdx` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179
<!-- 메인 이슈에 기록이 남아요! 가짜연구소 리포를 사용해 연습하실때는 제거해주시면 감사하겠습니다! :smile: -->

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

<!-- 제출 전 체크리스트로, 가짜연구소만의 체크리스트도 <details>로 감싸서 만들어두면 더 좋을 것 같아요. -->

## Who can review?
<!-- 가짜연구소 팀원들과 리뷰가 끝난 후에만 허깅페이스 직원들에게 리뷰 요청하는 아래 주석을 노출해주세요! -->
<!-- @sgugger, @ArthurZucker, @eunseojo May you please review this PR? -->

Team PseudoLab, may you please review this PR?

@0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

wonhyeongseo committed:
docs: ko: tasks/translation.mdx

wonhyeongseo commented on docs/source/ko/tasks/translation.mdx:
````diff
@@ -0,0 +1,405 @@
+>>> tokenized_books = books.map(preprocess_function, batched=True)
+```
+
+이제 [`DataCollatorForSeq2Seq`]를 사용하여 예제 배치를 생성합니다. 데이터세트의 최대 길이로 전부를 padding하는 대신, 데이터 정렬 중 해당 배치의 최대 길이로 문장을 *동적으로 padding*하는 것이 더 효율적입니다.
````
'해당 배치'를 '각 배치'로 하는 것이 더 자연스러울까요?

gabrielwithappy replied:
각 배치가 좀 더 장확할 것 같습니다. 배치별로 패딩길이를 다르게 할 수 있을이 나타날 것 같습니다

wonhyeongseo replied:
좋은 리뷰 정말 감사합니다! 반영되었어요 😆

wonhyeongseo commented on docs/source/ko/tasks/translation.mdx:
````diff
@@ -0,0 +1,405 @@
+[{'translation_text': 'Legumes partagent des ressources avec des bactéries azotantes.'}]
+```
+
+원한다면 `pipeline`의 결과를 직접 복제할 수도 있습니다.
````
결과를 복제한다는 의미가 무엇인지 잘 모르겠어요 😓 혹시 설명해주실 수 있을까요?

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_22678). All of your documentation changes will be reflected on that endpoint.

wonhyeongseo committed:
docs: ko: tasks/translation.mdx

docs: ko: tasks/translation.mdx

fix: this(해당) batch to each(각) batch

chore: wsl

docs: ko: tasks/translation.mdx

fix: this(해당) batch to each(각) batch

chore: wsl

KIHOON71 commented on docs/source/ko/tasks/translation.mdx:
```diff
@@ -0,0 +1,405 @@
+
+## 평가[[evalulate]]
+
+훈련 중에 메트릭을 포함하면 모델의 성능을 평가하는 데 도움이 됩니다. 🤗 [Evaluate](https://huggingface.co/docs/evaluate/index) 라이브러리로 평가 방법(evaluation method)을 빠르게 가져올 수 있습니다. 현재 태스크에 적합한 SacreBLEU 메트릭을 가져오세요. (메트릭을 가져오고 계산하는 방법에 대해 자세히 알아보려면 🤗 Evaluate [둘러보기](https://huggingface.co/docs/evaluate/a_quick_tour)를 참조하세요):
```
metric은 한국어로 '평가 지표' 나 '성능 지표' 등으로 대체할 수 있다고 생각하는데 괜찮을까요??

wonhyeongseo replied:
메트릭으로 번역된 부분이 많기 때문에 부득이하게 놔둔뒤, 나중에 한번에 바꿔보는 것을 고려해볼게요. 리뷰 감사합니다!

sim-so reviewed (changes requested):
항상 멋진 번역 감사합니다!
Draft 상태로 해두셨는데요, 리뷰를 해도 괜찮다면 계속 진행하겠습니다. 😄

wonhyeongseo commented on docs/source/ko/tasks/translation.mdx:
```diff
@@ -0,0 +1,405 @@
+
+<Youtube id="1JvfrvZgi6c"/>
+
+번역은 한 언어로 된 시퀀스를 다른 언어로 변환합니다. 받은 입력과 반환되는 출력으로 문제를 생각하는 강력한 프레임워크인 sequence-to-sequence 문제로 재구성할 수 있는 태스크 중 대표적인 예시가 번역이나 요약입니다. 번역 시스템은 일반적으로 다른 언어로 된 텍스트 간의 번역에 사용되지만, 음성 간의 통역이나 텍스트-음성 또는 음성-텍스트와 같은 조합에도 사용될 수 있습니다.
```
감사합니다! 반영되었습니다.

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_22678). All of your documentation changes will be reflected on that endpoint.

wonhyeongseo commented:
Closed in favor of https://github.com/huggingface/transformers/pull/22805
